Sources/App/CRP/JiraService+CRP.swift

```diff
@@ -20,7 +22,8 @@ extension JiraService {
 
     static func accountablePerson(release: GitHubService.Release) -> FieldType.User {
         let isTelus = release.appName.caseInsensitiveCompare("Telus") == .orderedSame
-        return isTelus ? .RyanCovill : .MarkBates
+        let isUS = release.appName.caseInsensitiveCompare("BabylonUS") == .orderedSame
+        return isTelus ? .RyanCovill : isUS ? .WilsonAmadi : .MarkBates
```

Comment:
[nit]
I think chaining if statements is a lot more readable than two ternarys

```suggestion
        if isTelus {
            return .RyanCovill
        } else if isUS {
            return .WilsonAmadi
        } else {
            return .MarkBates
        }
```

Reply:
🍝

Reply:
this shouldn't even be defined in code like this but it is what it is =)